scrapers/BelAmi.yml

```diff
@@ -0,0 +1,30 @@
+name: "BelAmi"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - belamionline.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //*[@id="ContentPlaceHolder1_LabelTitle"]
+      Details:
+        selector: //*[@id="ContentPlaceHolder1_LabelDescription"]|//div[@class='bottom']/p[2]
+        concat: " "
+      Date:
+        selector: //*[@id="ContentPlaceHolder1_LabelReleased"]
+        postProcess:
+          - parseDate: 1/02/2006
+      Studio:
+        Name: 
+          fixed: BelAmi
+      Performers:
+        Name: 
+          selector: //div[@class="actors_list"]//text()
+      Image:
+          selector: //div[@class="video_player"]//a/@href
```

Comment:
The selector needs to be more exact as it gets a lot more anchors
```yaml
        selector: //div[@class="video_player"]//a[contains(@href,"Screen-")]/@href
```
seems to get the single needed

Reply:
Confirmed and resolved.